src/iclienthints.cpp

```diff
@@ -4,6 +4,7 @@
 #include <irods/irods_client_api_table.hpp>
 #include <irods/irods_pack_table.hpp>
 #include <irods/rodsClient.h>
+#include <irods/rodsError.h>
```

Comment:
Is this needed?

Reply:
Yes. I had inserted a call to `printErrorStack` without including its header.